index.html

```diff
@@ -0,0 +1,193 @@
+
+<section class="section-3">
+   <div class="about-me">
+      <h1 class="h31">About me</h1>
+      <p class="p31">Hello I’m a software developer! I can help you build a product ,feature or 
+         website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <a class="get-myresume button" href="">Get My Resume</a>
+      <img class="abt-me-image" src="./assets/images/Group1.png" alt="">
+   </div>
+   <hr class="line-3">
+
+   <ul class="lists">
+      <li class="Lan">Language 
+          <ul class="lan-list">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+      </li>
+        
+      <li class="Frame"> Framework
+          <ul class="Frame-list">
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+          </ul>
+      </li>
+      <li class="Sk">Skills
+          <ul class="Sk-list">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+      </li>
+  </ul>
+</section>
+
+<form class="contact-form" action="https://formspree.io/f/{Portifolio-contact}" method="post">
+  <ul class="ul-contact">
+    <li class="p-contact ">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</li>
+    <li>
+      <label for="name"></label>
+      <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+    </li>
+    <li>
+      <label for="mail"></label>
+      <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+    </li>
+    <li>
+      <label for="msg"></label>
+      <textarea id="msg" name="user_message" placeholder="Enter text here" maxlength="500" required></textarea>
+    </li>
+    <li class="submit-button">
+      <button type="submit" id="get-in-touch">Get in touch</button>
+    </li>
+  </ul>
+
+ </ul>
+</form>
+
+<footer class="footer-social-media">
+  <ul class="social-media-list">
+      <li class="socia-c"><a href="https://github.com/solog0039"><img  src="./assets/social media footer/Icon -GitHub.png" alt="Github"></a></li>
+      <li class="social-c"><a href="www.linkedin.com/in/solomon-kidanu-62a994232"><img  src="./assets/social media footer/Icon - Linkedin.png" alt="linkedin"></a></li>
+      <li class="social-c"><a href="##"><img  src="./assets/social media footer/Angelist.png" alt="tum"></a></li>
+      <li class="social-c"><a href="https://twitter.com/Solomon57320119"><img  src="./assets/social media footer/Icon - Twitter.png" alt="tiwtter"></a></li>
+      <li class="social-c"><a href="##"><img  src="./assets/social media footer/medium.png" alt="Medium"></a></li>
+  </ul>
+  <hr class="line-c">
+</footer>
```

Comment:
Please remove the line circled red at the footer since its not part of the website, figma only used that to present how the portfolio looks on an iPhone screen. Also, try to address the padding issues in the footer especially the area pointed by the arrow to match the Figma template at least. Doing so will make the footer more organized.


SEE THE SCREENSHOT

<img width="227" alt="bad footer" src="https://user-images.githubusercontent.com/84930355/214453811-87070606-896d-4937-9453-c2fca9ed89a4.png">


FIGMA VERSION

<img width="330" alt="good foot" src="https://user-images.githubusercontent.com/84930355/214453877-cb533e40-cc13-4885-a7ad-84ad25dd43bd.png">